test/examples/py2pack-opensuse-augmented.spec

```diff
@@ -21,7 +21,7 @@ Name:           python-py2pack
 Version:        0.8.5
 Release:        0
 Summary:        Generate distribution packages from PyPI
-License:        UNKNOWN (FIXME:No SPDX)
```

Comment:
Why?

Reply:
Well, I don't really know why that changed now, also why the test fails in other parts of the specfile (see above)